README.rst

```diff
@@ -508,6 +509,40 @@ are supported:
 
 .. _cartridge-cli-packing-an-application:
 
+*********
+``clean``
+*********
+
+To remove instance(s) files (log, workdir, console and notify sockets),
+use the ``clean`` command:
+
+.. code-block:: bash
+
+    cartridge clean [INSTANCE_NAME...] [flags]
+
+By default, `cartridge clean` for running instances causes an error.
+But it can be forced with ``--force`` flag.
```

Comment:
I'm against it. We doesn't have to drop any data for running instances

Reply:
Yes, you are right. Removed.